API.md

```diff
@@ -69,62 +63,3 @@ The url is in the form:
 #### Example:
 
 http://geojson.io/#id=github:benbalter/dc-wifi-social/blob/master/bars.geojson&map=14/38.9140/-77.0302
-
-## Console API
-
-[Pop open your browser console](http://debugbrowser.com/) and see the beautiful
-examples: geojson.io has started to expose a subset of its inner workings for
-you to mess around with:
-
-
-### `window.api.map`
-
-The [Leaflet](http://leafletjs.com/) map that you see and use on the site. See
-the [Leaflet API](http://leafletjs.com/reference.html) for all the things you
-can do with it.
-
-For instance, you could add another map layer:
-
-```js
-window.api.map.addLayer(L.tileLayer('http://tile.stamen.com/watercolor/{z}/{x}/{y}.jpg'))
-```
-
-### `window.api.data`
-
-The data model. See the [code to get an idea of how it works](https://github.com/mapbox/geojson.io/blob/gh-pages/src/core/data.js#L48-L90) -
-you'll want to use stuff like `data.set({ map: { .. your geojson map information .. })`
-and `data.get('map')` and `data.mergeFeatures([arrayoffeatures])` to do your
-dirty business.
-
-## `window.api.mapLayer`
-
-This is the Leaflet featureGroup that gets filled with features as you draw
-them. You can operate on this directly to do advanced stuff like
-selecting a feature with its id:
-
-```js
-var layers = [];
-window.api.mapLayer.eachLayer(l => { layers.push(l); });
-layers.find(l => l.feature.id == 'a').openPopup();
-```
-
-That example uses [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
-which are supported in Chrome & Firefox.
-
-## `window.api.drawControl`
-
-Exposes the [Leaflet.Draw](https://github.com/Leaflet/Leaflet.draw) control
-instance in the console.
-
-## `window.api.on(event, fn)`
-
-Exposes d3 events, including `change`.
-
-## Protips
-
-To include `turf` from [turf](https://github.com/turfjs/turf) so you can manipulate features
```

Comment:
I never realized this part! Ah well . . .

Reply:
No longer relevant as we now need turf to do a bunch of stuff leaflet was doing.